package.json

```diff
@@ -30,32 +30,31 @@
     "build:theme-designer": "yarn workspace @twilio-paste/theme-designer build",
     "build:props": "lerna run build:props --ignore @twilio-paste/website --concurrency 2",
     "pre-push": "concurrently \"yarn:lint\" \"yarn:test\" \"yarn:prettier\" \"yarn:type-check\"",
-    "release": "yarn lerna publish -m 'chore(release): publish' --conventional-commits --canary --preid beta --dist-tag next",
```

Comment:
Does the new release script handle publishing the next tag now?

Reply:
Changesets can handle prereleases but it's a "whole thing". https://github.com/atlassian/changesets/blob/master/docs/prereleases.md

Much more involved and manual.

Reply:
Can we use the old pre-release script for next and the new script for full releases? It was a really nice feature, would be sad to lose it.

Reply:
I guess it should work... it only looks at git tags and npm so the theory is sound. I'll add it back in until we can test it with something

Reply:
OK back in under a `release:next` script @TheSisb